sw/device/silicon_creator/lib/drivers/pinmux.c

```diff
@@ -22,6 +22,11 @@ typedef struct pinmux_input {
  * Peripheral input to MIO pad mappings.
  */
 static const pinmux_input_t kPinmuxInputs[] = {
+    /**
+     * Bootstrap pin.
+     */
+    {.periph = kTopEarlgreyPinmuxPeripheralInGpioGpio17,
+     .pad = kTopEarlgreyPinmuxInselIob8},
```

Comment:
nit: comma after the last entry will probably make this look better.

Reply:
Done.